internal/remote/modulereleasemeta_syncer_test.go

```diff
@@ -35,6 +35,7 @@ func TestModuleReleaseMetaSyncer_SyncToSKR_happypath(t *testing.T) { //nolint:du
 	skrClient := fake.NewClientBuilder().
 		WithObjects(&mrmSKR2, &mrmSKR3, &mrmSKR4).
 		WithScheme(scheme).
+		WithReturnManagedFields().
```

Comment:
It seems there were changes to fakeClient. Without providing this flag, the List operation will not include Managed Fields which is required by this and the other tests.